drivers/media/i2c/ov5693.h

```diff
@@ -37,68 +37,23 @@
  */
 #define ENABLE_NON_PREVIEW	1
 
-#define OV5693_POWER_UP_RETRY_NUM 5
```

Comment:
How about merging this header with ov5693.c?

I noticed that newer sensor drivers don't use header files. Maybe to prevent duplication.

Reply:
I agree; that's on my list, but still a bit of trimming to do; wanted to get the header as small as possible before merging them...that file is loooooong already.